ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/zulu7/tasks/main.yml

```diff
@@ -28,7 +28,7 @@
   when:
     - ansible_architecture == "x86_64"
     - ((ansible_distribution == "Redhat" or ansible_distribution == "CentOS") and (ansible_distribution_major_version|int >= 8)) or
-      (ansible_distribution == "Ubuntu" and ansible_distribution_major_version|int >= 18) or
+      (ansible_distribution == "Ubuntu" and ansible_distribution_major_version|int >= 16) or
```

Comment:
Is there any reason to have the version check in there any more? Every version of Ubuntu we have will now need this right? (Alternative would be to just bootstrap using JDK8 of course but this solution is probably better :-) )

Reply:
Good point - I doubt anyone is going to be setting up an Ubuntu14 machine anytime soon :-)